### DISCUSSION TITLE
[ 问题咨询 ]如何实现同一个用户对不同实例有不同的权限

### DISCUSSION THREAD

**Discussion by dengdalue** [hhyo/Archery#1619] [Q&A]
### 问题描述
有两个实例db1、db2，用户xiaoming想要对db1有审核权限，对db2有执行权限，请问如何实现？

### 版本信息
 - 应用版本/分支：Release v1.7.7
 - 部署方式：Docker

**Comment by feiazifeiazi** [comment]
这个功能我也有需求。。很强烈。

**Comment by cslingjun** [comment]
db2是要对所有人提交的工单都有执行权限，还是自己提交的 ，如果是自己提交的话 ，分2个资源组就好了
